elan/rpc/gc.go

```diff
@@ -85,15 +85,13 @@ func (s *server) deleteAll(ctx context.Context, blobs []*ppb.Blob, hard bool) er
 		key := blob.CachePrefix + blob.Hash
 		if exists, err := s.bucket.Exists(ctx, key); exists {
 			if err := s.bucket.Delete(ctx, key, hard); err != nil {
-				me = multierror.Append(me, err)
+				me = multierror.Append(me, fmt.Errorf("Error deleting blob: %w", err))
 			}
 			if s.knownBlobCache != nil {
 				s.knownBlobCache.Del(key)
 			}
 		} else if err != nil {
-			me = multierror.Append(me, err)
-		} else {
-			me = multierror.Append(me, fmt.Errorf("Blob not found: %s", key))
```

Comment:
Why are we getting rid of this? I think this still makes sense. I'm happy if we change it to a `log.Error` instead though.

Reply:
I have logged this as a warning and not returned the error. The other option here is to return custom errors and handle them on  client side but that's annoying with multierrors.

Reply:
I think logging it makes sense, thanks! :+1: